webapp/src/main/resources/db/migration/V71__Add_scheduled_jobs.sql

```diff
@@ -0,0 +1,53 @@
+CREATE TABLE scheduled_job (
+    id CHAR(36) PRIMARY KEY,
+    repository_id bigint NOT NULL,
+    job_type smallint DEFAULT NULL,
+    cron varchar(50) DEFAULT NULL,
+    properties longtext NOT NULL,
+    job_status smallint NOT NULL,
```

Comment:
Nit: personally I prefer if all SQL keywords are uppercased. It makes it easier to mentally strip out boiler plate from user specific content

Reply:
eg ADD, TIMESTAMP, SMALLINT, VARCHAR, BIGINT etc